explorer/rest/tests/test_rest.py

```diff
@@ -68,6 +68,7 @@ def _assert_get_api_nem_block_by_height(client, height, expected_status_code, ex
 	# Assert:
 	assert expected_status_code == response.status_code
 	assert expected_result == response.json
+	assert response.headers['Access-Control-Allow-Origin'] == '*'
```

Comment:
consider helper function that tests status code + headers, so this is checked in all cases

e.g.
_assert_status_code_and_headers(response, 200)